oauth2client/service_account.py

```diff
@@ -461,3 +462,56 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+class SelfSignedJWTCredentials(object):
```

Comment:
Good point. In C++ we call them `ServiceAccountJWTAccessCredentials`. Would that answer your concern?